src/common/gui/CAboutBox.cpp

```diff
@@ -29,6 +35,26 @@ void CAboutBox::draw(CDrawContext* pContext)
    if (value)
    {
       _aboutBitmap->draw(pContext, getViewSize(), CPoint(0, 0), 0xff);
+
+      int strHeight = infoFont->getSize(); // There should really be a better API for this in VSTGUI
+      std::vector< std::string > msgs = { {
+              std::string( "Version 1.6.0 beta (build: " ) + __DATE__ + " " + __TIME__ + ")",
+              "Released under the GNU General Public License, v3",
+              "Copyright 2005-2019 by individual contributors",
+              "Source, contributors and other information at https://github.com/kurasu/surge",
+              "VST Plug-in technology by Steinberg, AU Plugin Technology by Apple Computer"
+          } };
+
+      int yMargin = 5;
+      int yPos = toDisplay.getHeight() - msgs.size() * (strHeight + yMargin); // one for the last; one for the margin
+      int xPos = strHeight;
+      pContext->setFontColor( kWhiteCColor );
+      pContext->setFont( infoFont );
+      for( auto s : msgs )
+      {
```

Comment:
Cosmetic: hinking about the white-spacing here and also in function calls. I use a different syntax. Which one Surge originally uses?

Reply:
You are 100% correct. That convention will never leave my fingers. I will go and fix it and squash in.

Reply:
OK I pushed this change. Thanks for the careful look!